patches/minecraft/net/minecraft/world/level/block/PointedDripstoneBlock.java.patch

```diff
@@ -0,0 +1,54 @@
+--- a/net/minecraft/world/level/block/PointedDripstoneBlock.java
++++ b/net/minecraft/world/level/block/PointedDripstoneBlock.java
+@@ -162,23 +_,12 @@
+ 
+    @VisibleForTesting
+    public static void m_221859_(BlockState p_221860_, ServerLevel p_221861_, BlockPos p_221862_, float p_221863_) {
+-      if (!(p_221863_ > 0.17578125F) || !(p_221863_ > 0.05859375F)) {
+          if (m_154203_(p_221860_, p_221861_, p_221862_)) {
+             Optional<PointedDripstoneBlock.FluidInfo> optional = m_154181_(p_221861_, p_221862_, p_221860_);
+             if (!optional.isEmpty()) {
+                Fluid fluid = (optional.get()).f_221893_;
+-               float f;
+-               if (fluid == Fluids.f_76193_) {
+-                  f = 0.17578125F;
+-               } else {
+-                  if (fluid != Fluids.f_76195_) {
+-                     return;
+-                  }
+-
+-                  f = 0.05859375F;
+-               }
```

Comment:
This doesn't need to be remove since it'll just go unused.

Reply:
I'll have to at least remove the part that makes it return if the fluid isn't lava since that will make the patch effectively useless, but I'll add back the other stuff